### ISSUE TITLE
Removal of missing package reports as successful

### ISSUE THREAD

**Issue by troydemonbreun** [yarnpkg/yarn#1408]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
When using `yarn remove [package]` and providing a package that is not in package.json, though nothing is removed, the results imply success:

```
[1/2] Removing module fassdfsdf...
[2/2] Regenerating lockfile and installing missing dependencies...
warning No license field
success Uninstalled packages.
```

**If the current behavior is a bug, please provide the steps to reproduce.**
`yarn remove [package]` where package is not in package.json.

**What is the expected behavior?**
Error to indicate a no-op, such as:

```
[1/2] Removing module fassdfsdf...
error This module isn't specified in a manifest.
```

**Please mention your node.js, yarn and operating system version.**
yarn v0.16.2
node v6.5.0
windows 10

**Comment by troydemonbreun** [comment]
@wyze This is functionality that basically already existed (already had a `reporter.lang` key, `found` flag, etc.).  It appears that the IF was just not specific enough to flip the `found` flag.

**Comment by kittens** [comment]
Fixed via #1409.
